packages/core/src/utils-hoist/severity.ts

```diff
@@ -19,5 +12,7 @@ export const validSeverityLevels = ['fatal', 'error', 'warning', 'log', 'info',
  * @returns The `SeverityLevel` corresponding to the given string, or 'log' if the string isn't a valid level.
  */
 export function severityLevelFromString(level: SeverityLevel | string): SeverityLevel {
-  return (level === 'warn' ? 'warning' : validSeverityLevels.includes(level) ? level : 'log') as SeverityLevel;
+  return (
+    level === 'warn' ? 'warning' : ['fatal', 'error', 'warning', 'log', 'info', 'debug'].includes(level) ? level : 'log'
+  ) as SeverityLevel;
```

Comment:
```suggestion
  ) satisfies SeverityLevel;
```

i think this is a bit less invasive? 🤔

Reply:
sadly doesn't work since `string` (which is the type that is passed in) does not satisfy `SeverityLevel`

Reply:
sad!